packages/bootstrap/docs/index.md

```diff
@@ -38,7 +34,7 @@ If you include the Bootstrap framework from its source, you have to add `kendo-t
 
 ## Preview
 
-To apply the Bootstrap theme to any of the examples on this website, use the theme chooser drop-down at the top-right corner of the page. The Material theme styling is loaded by default.
+To apply the Bootstrap theme to any of the examples on this website, use the theme chooser drop-down at the top-right corner of the example. {% platform_content angular %}The Default theme styling is loaded by default.{% endplatform_content %} The Material theme styling is loaded by default.
```

Comment:
This will render incorrectly on the Angular site as:
`The Default theme styling is loaded by default. The Material theme styling is loaded by default.`
I think there should be a platform condition added for react and vue for the second sentence.